paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/action/CloneActionITCase.java

```diff
@@ -0,0 +1,400 @@
+                });
+
+        /** write data into source table and commit them */
+        Thread writeDataToSourceTablethread =
+                new Thread(
+                        () -> {
+                            try {
+                                /** write and commit data to source table every 100 ms. */
+                                while (true) {
+                                    writeData(
+                                            rowData(
+                                                    1,
+                                                    100,
+                                                    15,
+                                                    BinaryString.fromString("20221208")),
+                                            rowData(
+                                                    1,
+                                                    100,
+                                                    16,
+                                                    BinaryString.fromString("20221208")),
+                                            rowData(
+                                                    1,
+                                                    100,
+                                                    15,
+                                                    BinaryString.fromString("20221209")),
+                                            rowData(
+                                                    2,
+                                                    100,
+                                                    15,
+                                                    BinaryString.fromString("20221208")),
+                                            rowData(
+                                                    2,
+                                                    100,
+                                                    16,
+                                                    BinaryString.fromString("20221208")),
+                                            rowData(
+                                                    2,
+                                                    100,
+                                                    15,
+                                                    BinaryString.fromString("20221209")));
+                                }
+                            } catch (Exception ignored) {
+                            }
+                        });
+        writeDataToSourceTablethread.start();
+
+        /** do clone */
+        int runCloneJobCount = 0;
+        boolean cloneJobHasException = false;
+        do {
+            try {
+                ++runCloneJobCount;
+                runCloneLatestSnapshotAction();
+                cloneJobHasException = false;
+            } catch (Exception e) {
+                cloneJobHasException = true;
+            }
+        } while (cloneJobHasException);
+
+        Assertions.assertTrue(runCloneJobCount > 1);
```

Comment:
Why do you check this? Why don't you check clone result?

Reply:
done